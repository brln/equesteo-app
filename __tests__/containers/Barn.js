import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import BarnContainer from '../../containers/Barn'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    localState: {},
    pouchRecords: {
      horses: {},
      horseUsers: {}
    }
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <BarnContainer/>
    </Provider>).toJSON()
  expect(tree).toMatchSnapshot()
})

