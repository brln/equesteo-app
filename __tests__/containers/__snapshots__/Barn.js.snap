// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTScrollView>
    <View>
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "flex": 1,
            "flexDirection": "row",
            "flexWrap": "wrap",
            "justifyContent": "space-between",
            "marginBottom": 30,
            "marginLeft": 10,
            "marginRight": 10,
            "marginTop": 30,
          }
        }
      >
        <View
          elevation={5}
          style={
            Object {
              "backgroundColor": "#5bc0eb",
              "marginBottom": 20,
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 3,
                "width": 0,
              },
              "shadowOpacity": 1,
              "shadowRadius": 5,
            }
          }
        >
          <View
            accessible={true}
            clickable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
                "padding": 5,
              }
            }
          >
            <View
              style={
                Object {
                  "borderColor": "transparent",
                  "borderRadius": 0,
                  "borderWidth": 2,
                  "height": 312.5,
                  "width": 312.5,
                }
              }
            >
              <Image
                source={1}
                style={
                  Object {
                    "borderRadius": 0,
                    "height": "100%",
                    "width": "100%",
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "bottom": 5,
                  "justifyContent": "center",
                  "left": 5,
                  "padding": 5,
                  "position": "absolute",
                  "right": 5,
                  "top": 5,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "white",
                    "textAlign": "center",
                    "textShadowColor": "black",
                    "textShadowOffset": Object {
                      "height": 1,
                      "width": -1,
                    },
                    "textShadowRadius": 5,
                  }
                }
              >
                Add New Horse
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
